SPECS/curl/curl.spec

```diff
@@ -1,15 +1,14 @@
 Summary:        An URL retrieval utility and library
 Name:           curl
 # Heads up: 7.87 breaks perl-WWW-Curl (see #4588).
-Version:        7.88.1
-Release:        2%{?dist}
+Version:        8.2.1
```

Comment:
Double checking here.
I faintly recall bumping curl anymore than 7.88.1 broke a few packages (I think perl-*).
It would be a good idea to verify the full build before merging.

Reply:
Running a full build to verify:
AMD64: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=408827&view=results
ARM64: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=408861&view=results